### PR TITLE
WIP TSK-1289: Upgrade to Angular 9

### PR DESCRIPTION
**Pull request by cnguyen-de**
Due to bootstrap issues, it is only possible to upgrade to Angular 9. Upgrading to Angular 10 will be in another PR

Please test all components extensively, this PR will be updated again when all Frontend PRs are merged

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -77,11 +77,11 @@
     "karma-jasmine-html-reporter": "1.4.0",
     "moment": "2.22.1",
     "ng2-mock-component": "0.1.1",
-    "prettier": "^2.0.5",
```
Comment (Tristan2357) [inline web/package.json:80]: Why did you remove the "^" here, but kept it in other places?

**Comment by cnguyen-de** [comment]
new PR will be created
